chess-AI/experimental-work/NN Layout.py

```diff
@@ -0,0 +1,83 @@
+
+
+# this is the basic AI CNN layout
+# we will use this and improve it until it can be used for the final product.
+
+
+import torch
+import numpy as np
+from torch import nn
+import torch.nn.functional as F
+import torchvision
```

Comment:
Can clean up these imports; 1) torchvision and numpy don't seem to be used 2) if you `import torch`, you don't need `from torch import nn`, can just do `torch.nn` when you need to access, same thing with `torch.nn.functional`. instead of `import torch`, maybe you can do `from torch import nn, randn`